libbeat/processors/add_host_metadata/add_host_metadata.go

```diff
@@ -10,15 +10,20 @@ import (
 
 	"github.com/elastic/beats/libbeat/beat"
 	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/feature"
 	"github.com/elastic/beats/libbeat/logp"
 	"github.com/elastic/beats/libbeat/processors"
 	"github.com/elastic/go-sysinfo"
 	"github.com/elastic/go-sysinfo/types"
 )
 
-func init() {
-	processors.RegisterPlugin("add_host_metadata", newHostMetadataProcessor)
-}
+const (
+	processorName   = "add_host_metadata"
+	cacheExpiration = time.Minute * 5
+)
+
+// Feature exposes add_host_metadata.
+var Feature = processors.Feature(processorName, newHostMetadataProcessor, feature.Stable)
```

Comment:
Is this variable here used for something?

Reply:
Yes this is used by the global registry, see comment https://github.com/elastic/beats/pull/7392#issuecomment-399204275 for a bit more details about the usage.